authors/apps/authentication/tests/basetests.py

```diff
@@ -24,17 +24,21 @@ def setUp(self):
         self.register_url = reverse("authentication:registration")
         self.update_url = reverse("authentication:update_get")
         self.get_url = reverse("authentication:update_get")
+        # self.verification_of_user = reverse("authentication:signup_verification")
```

Comment:
Kindly remove commented out code

Reply:
Yes, let me create a bug fix to remove those commented code